### PR TITLE
Feature/Frontend visible no auth

### PR DESCRIPTION
**Pull request by yeraycat**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <template v-if="isAppReady">
-    <template v-if="isLoggedIn">
+    <template v-if="!hideNav">
```
Comment (ArtSuslov) [inline src/App.vue:3]: boolean variables should be prefixed with 'is'

File: src/components/Nav.vue [hunk 2]
```diff
@@ -50,7 +50,7 @@
         <span>IBCs</span>
       </router-link>
     </div>
-    <div class="nav__activities">
+    <div class="nav__activities" v-if="isLoggedIn">
```
Comment (ArtSuslov) [inline src/components/Nav.vue:53]: v-if should go [first](https://v2.vuejs.org/v2/style-guide/?redirect=true#Element-attribute-order-recommended)

File: src/components/SortLine.vue [hunk 3]
```diff
@@ -69,10 +69,12 @@ import InputField from '@/components/fields/InputField.vue'
 import SearchIcon from '@/components/icons/SearchIcon.vue'
 import {
   sortingActivities,
-  sortingOwners,
+  getSortingOwners,
   ACTIVITIES_SORT,
   OWNERS_SORT,
 } from '@/helpers/sortingHelpers'
+// import { useAuthorization } from '@/composables/useAuthorization'
```
Comment (ArtSuslov) [inline src/components/SortLine.vue:76]: comments should be meaningful or removed

File: src/components/SortLine.vue [hunk 4]
```diff
@@ -87,6 +89,8 @@ export default defineComponent({
     const sortByActivites = ref(ACTIVITIES_SORT.latest)
     const sortByOwners = ref(OWNERS_SORT.all)
     const searchValue = ref('')
+    const walletAddress = wallet.isEmpty ? ' ' : wallet.account.address
```
Comment (ArtSuslov) [inline src/components/SortLine.vue:92]: is it okay to use non-empty string in this case?

File: src/components/SortLine.vue [hunk 5]
```diff
@@ -114,6 +118,8 @@ export default defineComponent({
       sortByActivites,
       sortByOwners,
       searchValue,
+      walletAddress,
```
Comment (ArtSuslov) [inline src/components/SortLine.vue:121]: not being used, no need to return

File: src/components/ValidatorInfo.vue [hunk 6]
```diff
@@ -95,7 +98,10 @@
             </div>
           </div>
         </div>
-        <div class="validator-info__delegetion-btn-wrapper">
+        <div
+          class="validator-info__delegetion-btn-wrapper"
+          v-if="accountAddress"
```
Comment (ArtSuslov) [inline src/components/ValidatorInfo.vue:103]: !

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: src/components/ValidatorsTableRow.vue [hunk 7]
```diff
@@ -130,6 +130,7 @@ export default defineComponent({
     },
     tabStatus: { type: String, required: true },
     inactiveValidatorsTitle: { type: String, required: true },
+    showActionButtons: { type: Boolean, required: true },
```
[the remark this answers is not in the document]
Comment (yeraycat) [reply src/components/ValidatorsTableRow.vue:133]: Can I use `has` instead of `is`?

File: src/components/SortLine.vue (hunk 4 above)
Comment (yeraycat) [reply src/components/SortLine.vue]: It was a typo I think

File: src/views/Requests.vue [hunk 8]
```diff
@@ -101,7 +102,7 @@
       />
     </template>
 
-    <div class="view-main__mobile-activities">
+    <div class="view-main__mobile-activities" v-if="accountAddress">
```
Comment (nikita1tup) [inline src/views/Requests.vue:105]: v-if should go first

Comment (yeraycat) [reply src/views/Requests.vue]: Yep, sorry for that. Fixed.

**Pull request merged**